package.json

```
@@ -23,7 +23,8 @@
     "CHANGELOG.md",
     "LICENSE",
     "readme.md",
-    "lib"
+    "lib",
+    "index.d.ts"
```

minor, optional: I would probably have reversed these:

```suggestion
    "index.d.ts",
    "lib"
```

- keep the non-documentation files alphabetical
- keep the interface file between the documentation and executable files

but this is just personal preference